src/patternfly/components/BackgroundImage/styles.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../patternfly-utilities";
+
+.pf-c-background-image {
+  &::before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: -1;
+    width: 100vw;
+    height: 100vh;
+    content: "";
+    background: url("#{$pf-global--image-path}/pfbg_576.jpg") no-repeat;
+    filter: url("#{$pf-global--image-path}/background-filter.svg#image_overlay");
+    background-attachment: fixed;
```

Comment:
how about adding `background-size: cover` and `background-position: center center;`?

Reply:
We still don't know if this will be a full page background or not.